LeezeIt/LeezeIt/ViewController.swift

```diff
@@ -7,13 +7,131 @@
 
 import UIKit
 
-class ViewController: UIViewController {
 
+
+class ViewController: UIViewController {
+    
+    let parentStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.distribution = .fillEqually
+        stackView.axis = .vertical
+        stackView.spacing = 190
+        return stackView
+    }()
+    
+    let logoImageStackView:  UIStackView = {
+        let stackView = UIStackView()
+        return stackView
+        
+    }()
+    
+    let buttonStackView:  UIStackView = {
+        let stackView = UIStackView()
+        stackView.distribution = .fillEqually
+        stackView.axis = .vertical
+        stackView.spacing = 30
+        return stackView
+    }()
+    
+    var logoImage: UIImageView = {
+        let lztImage =  UIImageView(image: .init(named: "L"))
+        lztImage.contentMode                               = .scaleToFill
+        lztImage.clipsToBounds                             = true
+        return lztImage
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        view.addSubview(parentStackView)
+        parentStackViewConst()
+        
+    } 
+    
+
+
+    
+    @objc  func logIn(_ sender: UIButton) {
+        print("LogIn Button Tapped")
+        
+    }
+    
+    @objc  func signUP(_ sender: UIButton) {
+        print("signUp Button Tapped")
+        
     }
+}
 
 
+extension ViewController {
```

Comment:
don't write short forms, write Constraints instead of Const

Reply:
resolved

Reply:
the image that you have in the assets, change the name of that "L" isn't a good name